README.md

```diff
@@ -1 +1,16 @@
-# hello-mern-wordle
\ No newline at end of file
+# wordle-lite
+
+## Developer Environment
+
+* For the backend service, run "cd backend && yarn install && yarn dev".  
+
+* For the frontend service, run "cd frontend && npm install && npm run dev".
```

Comment:
Good point!  I'd like to sync-up with you on this one to understand my options here.  The reason they are different now, is because I stole from the Frontend Master's tutorials (two different courses heehee).  
- [ ] Discuss package manager aggregation options